### PR TITLE
set up fetch!

### PR DESCRIPTION
**Pull request by jackdavidweber**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -28,6 +28,9 @@
     <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
     <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
 
+    <!-- week2 JS fetching -->
+    <script type="text/javascript" src="script.js"></script>
+
```
Comment (Danish21) [inline portfolio/src/main/webapp/index.html:33]: Nit:
Extra new line compared to other spacing in this file.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -30,3 +30,11 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+console.log("hello world");
```
Comment (Danish21) [inline portfolio/src/main/webapp/script.js]: Remove console.log() statement before merging.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -30,3 +30,11 @@ function addRandomQuote() {
...
+
+async function getRandomQuoteUsingAsyncAwait() {
```
Comment (Danish21) [inline portfolio/src/main/webapp/script.js]: Uber Nit:
I suggest changing to just getRandomQuote(). The caller of the function should not care about the implementation.

**Review by Danish21 — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jackdavidweber) [reply portfolio/src/main/webapp/index.html:33]: fixed!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (jackdavidweber) [reply portfolio/src/main/webapp/script.js]: got it!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (jackdavidweber) [reply portfolio/src/main/webapp/script.js]: makes sense! done!

**Pull request merged**